src/theme/theme.ts

```diff
@@ -183,4 +183,14 @@ export const theme = createTheme({
       lineHeight: '18px',
     },
   },
+  components: {
+    MuiAutocomplete: {
+      styleOverrides: {
+        option: {
+          height: '32px !important',
+          minHeight: '32px !important', //minHeight required here because in selector component, props className MuiAutocomplete-options overrides by applying minWidth : 48px in smaller screens
+        },
+      },
+    },
+  },
 })
```

Comment:
I think it's not a good idea to override the autocomplete style from the theme. We should be able to do this using css selectors.